src/app/routes/onDemandAudio/podcastExternalLinks/tamil.ts

```diff
@@ -0,0 +1,17 @@
+const externalLinks = {
+  p0l97701: [
+    {
+      linkText: 'Spotify',
+      linkUrl: 'https://open.spotify.com/show/3zmt7JkaUld5DltiSMuPQ6',
+      linkType: 'spotify',
+    },
+    {
+      linkText: 'Apple',
+      linkUrl:
+        'https://podcasts.apple.com/in/podcast/%E0%AE%89%E0%AE%B2%E0%AE%95-%E0%AE%A9-%E0%AE%95%E0%AE%A4/id1813372268',
```

Comment:
Please note, I've used the `in` country code, instead of `au` which was requested in the ticket...
```suggestion
        'https://podcasts.apple.com/in/podcast/%E0%AE%89%E0%AE%B2%E0%AE%95-%E0%AE%A9-%E0%AE%95%E0%AE%A4/id1813372268',
```